results/similar_developers_for_spadini.davide@gmail.com.json

```diff
@@ -0,0 +1,250 @@
+{
+    "a.serban@cs.ru.nl": {
```

Comment:
let's add short description to readme with results and explanation why these developers are similar. For me it's not clear why the most similar developer has no identifiers